### ISSUE TITLE
npm ci run failing in docker

### ISSUE THREAD

**Issue by 0mnius** [wip/action#26]
Hello,

The action is failing on npm ci run.

```
2020-10-25T11:19:06.1790805Z npm notice 
2020-10-25T11:19:06.1791266Z npm notice New patch version of npm available! 7.0.3 -> 7.0.5
2020-10-25T11:19:06.1791901Z npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.5>
2020-10-25T11:19:06.1792502Z npm notice Run `npm install -g npm@7.0.5` to update!
2020-10-25T11:19:06.1792845Z npm notice 
2020-10-25T11:19:06.1799788Z npm ERR! The `npm ci` command can only install with an existing package-lock.json or
2020-10-25T11:19:06.1800615Z npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
2020-10-25T11:19:06.1801376Z npm ERR! later to generate a package-lock.json file, then try again.
2020-10-25T11:19:06.1884172Z npm ERR! A complete log of this run can be found in:
2020-10-25T11:19:06.1885100Z npm ERR!     /root/.npm/_logs/2020-10-25T11_19_06_180Z-debug.log
2020-10-25T11:19:06.3839692Z The command '/bin/sh -c npm ci --only=production' returned a non-zero code: 1
2020-10-25T11:19:06.3864834Z ##[warning]Docker build failed with exit code 1, back off 3.163 seconds before retry.
```

Anything i can do to fix it on my end?

TIA,
Eli

**Comment by gr2m** [comment]
I don't think that this is a problem with with this action? What does your workflow file look like?

**Comment by 0mnius** [comment]
Hey Gregor,

Thanks for getting back so quickly.
The workflow is fairly straightforward:

```
name: '_style-wip-check'

on:
  - pull_request

jobs:
  style-wip-check:
    runs-on: ubuntu-latest
    if: ${{ ! startsWith(github.head_ref, 'changeset-release/') }}

    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

    steps:
      - name: 'commit message check for WIPs'
        uses: wip/action@v1.0.0
```

**Comment by shiwano** [comment]
We have the same problem.
Our workflow file is here.

```
name: WIP
on:
  pull_request:
    types: [ opened, synchronize, reopened, edited ]
jobs:
  wip:
    runs-on: ubuntu-latest
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - uses: wip/action@v1.0.0
```

**Comment by bhekanik** [comment]
We're also experiencing the same problem

**Comment by gr2m** [comment]
Okay thank you all for letting me know. Likely GitHub changed something, and it's time to upgrade the the action to the format. I'll look into it

**Comment by TGTGamer** [comment]
Additional logging data from my workflow: 

workflow config
```yml
name: Pull Request Management
on:
  pull_request:
    types:
      - opened
      - edited
      - synchronize
jobs:
  # Automatically blocks WIP pull request based on the title
  wip:
    runs-on: ubuntu-latest
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - name: Get Configurations
        uses: Videndum/manage-github-secrets@1.0.0-beta
        with:
          settings: ${{ secrets.SETTINGS }}
          mode: 'environment'
          token: ${{ secrets.GITHUB_TOKEN }}
      # - name: Work in progress label
      #   uses: y-temp4/add-wip-label@v1.0.0
      #   if: ${{env.wip_label}} == true
      #   with:
      #     repo-token: ${{ secrets.GITHUB_TOKEN }}
      - name: Work in progress status
        uses: wip/action@v1.0.0
        if: ${{env.wip_enabled}} == true
```

Console output (debug active)
```cmd
25s
##[debug***Evaluating condition for step: 'Build wip/action@v1.0.0'
##[debug***Evaluating: success()
##[debug***Evaluating success:
##[debug***=> true
##[debug***Result: true
##[debug***Starting: Build wip/action@v1.0.0
##[group***Build container for action use: '/home/runner/work/_actions/wip/action/v1.0.0/Dockerfile'.
##[command***/usr/bin/docker build -t 1e5c35:86a5ee67b9944feca981eea4bbd44c9d -f "/home/runner/work/_actions/wip/action/v1.0.0/Dockerfile" "/home/runner/work/_actions/wip/action/v1.0.0"
Sending build context to Docker daemon  287.7kB

Step 1/12 : FROM node:slim
slim: Pulling from library/node
babf97a3f00a: Pulling fs layer
d81eb1809d95: Pulling fs layer
dbaab951a9fa: Pulling fs layer
cb915c1b28b9: Pulling fs layer
7c72cd70a544: Pulling fs layer
cb915c1b28b9: Waiting
7c72cd70a544: Waiting
d81eb1809d95: Verifying Checksum
d81eb1809d95: Download complete
babf97a3f00a: Verifying Checksum
babf97a3f00a: Download complete
cb915c1b28b9: Verifying Checksum
cb915c1b28b9: Download complete
7c72cd70a544: Verifying Checksum
7c72cd70a544: Download complete
dbaab951a9fa: Verifying Checksum
dbaab951a9fa: Download complete
babf97a3f00a: Pull complete
d81eb1809d95: Pull complete
dbaab951a9fa: Pull complete
cb915c1b28b9: Pull complete
7c72cd70a544: Pull complete
Digest: sha256:bf39db19ac53a635030d5db569cab912322147139fbf929c252359f3c50c4207
Status: Downloaded newer image for node:slim
 ---> 922b09b21278
Step 2/12 : LABEL "com.github.actions.name"="WIP"
 ---> Running in d8e547135231
Removing intermediate container d8e547135231
 ---> fcd39c3ac681
Step 3/12 : LABEL "com.github.actions.description"="Set WIP pull requests to pending"
 ---> Running in f211cfa35dfd
Removing intermediate container f211cfa35dfd
 ---> 34a3ecc95acc
Step 4/12 : LABEL "com.github.actions.icon"="edit"
 ---> Running in 5a646f2522ba
Removing intermediate container 5a646f2522ba
 ---> ce2e35ba2ac5
Step 5/12 : LABEL "com.github.actions.color"="orange"
 ---> Running in e00426dbd127
Removing intermediate container e00426dbd127
 ---> a7690d4b1123
Step 6/12 : LABEL "repository"="http://github.com/wip/action"
 ---> Running in 1760433dacf7
Removing intermediate container 1760433dacf7
 ---> 6032471542c8
Step 7/12 : LABEL "homepage"="http://github.com/wip/action"
 ---> Running in cfc149d35d51
Removing intermediate container cfc149d35d51
 ---> 82b3ca61adca
Step 8/12 : LABEL "maintainer"="Gregor Martynus (https://github.com/gr2m)"
 ---> Running in 7609663f5fdb
Removing intermediate container 7609663f5fdb
 ---> 2ba24a41fc80
Step 9/12 : COPY package*.json ./
 ---> c1d4ea8e8faa
Step 10/12 : RUN npm ci --only=production
 ---> Running in 89e8a01c803f
npm notice 
npm notice New patch version of npm available! 7.0.3 -> 7.0.7
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.7>
npm notice Run `npm install -g npm@7.0.7` to update!
npm notice 
npm ERR! The `npm ci` command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-03T04_33_29_115Z-debug.log
The command '/bin/sh -c npm ci --only=production' returned a non-zero code: 1

##[warning***Docker build failed with exit code 1, back off 4.78 seconds before retry.
##[command***/usr/bin/docker build -t 1e5c35:86a5ee67b9944feca981eea4bbd44c9d -f "/home/runner/work/_actions/wip/action/v1.0.0/Dockerfile" "/home/runner/work/_actions/wip/action/v1.0.0"
Sending build context to Docker daemon  287.7kB

Step 1/12 : FROM node:slim
 ---> 922b09b21278
Step 2/12 : LABEL "com.github.actions.name"="WIP"
 ---> Using cache
 ---> fcd39c3ac681
Step 3/12 : LABEL "com.github.actions.description"="Set WIP pull requests to pending"
 ---> Using cache
 ---> 34a3ecc95acc
Step 4/12 : LABEL "com.github.actions.icon"="edit"
 ---> Using cache
 ---> ce2e35ba2ac5
Step 5/12 : LABEL "com.github.actions.color"="orange"
 ---> Using cache
 ---> a7690d4b1123
Step 6/12 : LABEL "repository"="http://github.com/wip/action"
 ---> Using cache
 ---> 6032471542c8
Step 7/12 : LABEL "homepage"="http://github.com/wip/action"
 ---> Using cache
 ---> 82b3ca61adca
Step 8/12 : LABEL "maintainer"="Gregor Martynus (https://github.com/gr2m)"
 ---> Using cache
 ---> 2ba24a41fc80
Step 9/12 : COPY package*.json ./
 ---> Using cache
 ---> c1d4ea8e8faa
Step 10/12 : RUN npm ci --only=production
 ---> Running in 1c5c0cb20960
npm notice 
npm notice New patch version of npm available! 7.0.3 -> 7.0.7
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.7>
npm notice Run `npm install -g npm@7.0.7` to update!
npm notice 
npm ERR! The `npm ci` command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-03T04_33_35_273Z-debug.log
The command '/bin/sh -c npm ci --only=production' returned a non-zero code: 1

##[warning***Docker build failed with exit code 1, back off 1.935 seconds before retry.
##[command***/usr/bin/docker build -t 1e5c35:86a5ee67b9944feca981eea4bbd44c9d -f "/home/runner/work/_actions/wip/action/v1.0.0/Dockerfile" "/home/runner/work/_actions/wip/action/v1.0.0"
Sending build context to Docker daemon  287.7kB

Step 1/12 : FROM node:slim
 ---> 922b09b21278
Step 2/12 : LABEL "com.github.actions.name"="WIP"
 ---> Using cache
 ---> fcd39c3ac681
Step 3/12 : LABEL "com.github.actions.description"="Set WIP pull requests to pending"
 ---> Using cache
 ---> 34a3ecc95acc
Step 4/12 : LABEL "com.github.actions.icon"="edit"
 ---> Using cache
 ---> ce2e35ba2ac5
Step 5/12 : LABEL "com.github.actions.color"="orange"
 ---> Using cache
 ---> a7690d4b1123
Step 6/12 : LABEL "repository"="http://github.com/wip/action"
 ---> Using cache
 ---> 6032471542c8
Step 7/12 : LABEL "homepage"="http://github.com/wip/action"
 ---> Using cache
 ---> 82b3ca61adca
Step 8/12 : LABEL "maintainer"="Gregor Martynus (https://github.com/gr2m)"
 ---> Using cache
 ---> 2ba24a41fc80
Step 9/12 : COPY package*.json ./
 ---> Using cache
 ---> c1d4ea8e8faa
Step 10/12 : RUN npm ci --only=production
 ---> Running in 606f4069ffb5
npm notice 
npm notice New patch version of npm available! 7.0.3 -> 7.0.7
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.0.7>
npm notice Run `npm install -g npm@7.0.7` to update!
npm notice 
npm ERR! The `npm ci` command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-03T04_33_38_560Z-debug.log
The command '/bin/sh -c npm ci --only=production' returned a non-zero code: 1

##[endgroup***
##[error***Docker build failed with exit code 1
##[debug***System.InvalidOperationException: Docker build failed with exit code 1
##[debug***   at GitHub.Runner.Worker.ActionManager.BuildActionContainerAsync(IExecutionContext executionContext, Object data)
##[debug***   at GitHub.Runner.Worker.JobExtensionRunner.RunAsync()
##[debug***   at GitHub.Runner.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken)
##[debug***Finishing: Build wip/action@v1.0.0
```

**Comment by gr2m** [comment]
releases are now automated and are continuously pushed to the `v1` branch. So you can do 

```yml
- uses wip/action@v1
```
